test/test_clova.py

```diff
@@ -125,6 +134,11 @@ def test_event_handler(self):
 
         self.assertIsNone(response)
 
+    def test_event_handler_audio_player(self):
```

Comment:
Maybe I missed something but it's not clear to me what is actually tested here.

Reply:
It would test if a response is returned from the handler which shouldn't be the case. It's more testing the test implementation. You are right. There isn't that much sense to test this. Same for the `test_event_handler`

Reply:
After closer investigation, it might be better todo a separate p-r to restructure the tests and maybe this among others a bit more meaning full.